### PR TITLE
Add command line option for "reopen"

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/113611

### CODE REVIEW

**Review by kalikiana — APPROVED** [review]
Nice and simple.

File: openqa_review/openqa_review.py [hunk 1]
```diff
@@ -1628,7 +1633,8 @@ def reminder_comment_on_issue(ie, args):
         comment = openqa_issue_comment.substitute(
             {"name": f["name"], "url": urljoin(ie._url(f), link_to_module), "time_next": next_threshold}
         ).strip()
-        issue.reopen(comment)
+        if args.reopen or args.reopen == issue.issue_type:
```
Comment (Martchus) [inline openqa_review/openqa_review.py:1636]: I'm not quite sure what's wanted her but are you sure the "or" is correct? I mean if reopen is truthy then the equality check shouldn't matter anymore anyways.

File: openqa_review/openqa_review.py [hunk 2]
```diff
@@ -1444,6 +1444,11 @@ def parse_args():
         default=NO_REMINDER_REGEX,
         help="""A regular expression which will suppress sending the reminder on match.""",
     )
+    reminder_comments.add_argument(
+        "--reopen",
+        default="redmine",
+        help="""En-/disable reopening closed issues or select only which issue types should be reopened, e.g 'redmine' or 'bugzilla'.""",
```
Comment (Martchus) [inline openqa_review/openqa_review.py:1450]: What would one pass to disable? And have you tested that the argument parser will really  make a falsy value?

File: openqa_review/openqa_review.py (hunk 1 above)
Comment (okurz) [reply openqa_review/openqa_review.py]: done

File: openqa_review/openqa_review.py (hunk 2 above)
Comment (okurz) [reply openqa_review/openqa_review.py]: done

**Review by Martchus — APPROVED** [review]

**Pull request merged**